md_api/src/main/java/com/wipro/ats/bdre/md/api/ProcessLog.java

```diff
@@ -151,6 +152,31 @@ public void log(ProcessLogInfo processLogInfo) {
         }
 
     }
+
+    public void logList(List<ProcessLogInfo> processLogInfo) {
+        try {
+
+            //calling addprocesslog function of Addprocesslogdao
+            processLogDAO.logList(processLogInfo);
+
+        } catch (Exception e) {
+            LOGGER.error("Error occurred.", e);
+            throw new MetadataException(e);
+        }
+
+    }
+
+    public List<ProcessLogInfo> listLastInstanceRef(int processId, String messageId) {
+        try {
+
+            //calling addprocesslog function of Addprocesslogdao
+            List<ProcessLogInfo> processLogInfos=processLogDAO.listLastInstanceRef(processId,messageId);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "processLogInfos". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1488)